1.cellprofiler_ic/scripts/nf1_ic.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Correct illumination and save images for each plate using CellProfiler
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+
+import utils.cp_utils as cp_utils
+
+
+# ## Set paths for each plate
+
+# In[2]:
+
+
+path_to_pipeline = pathlib.Path("NF1_illum.cppipe").absolute()
+
+plates_info_dictionary = {
+    "Plate_1": {
+        "path_to_images": pathlib.Path("../0.download_data/Plate_1/").absolute(),
+        "path_to_output": pathlib.Path("Corrected_Plate_1").absolute(),
+    },
+    "Plate_2": {
+        "path_to_images": pathlib.Path("../0.download_data/Plate_2/").absolute(),
+        "path_to_output": pathlib.Path("Corrected_Plate_2").absolute(),
+    },
+}
+
+
+# ## Run illumination correction pipeline on each plate
+
+# In[3]:
+
+
+for plate in plates_info_dictionary:
```

Comment:
Consider using `plates_info_dictionary.items()` with this for loop to access both the key and value without the bracketed notation below.

Reply:
Asked about this to you and it is now implemented!